### DISCUSSION TITLE
我的專案裡的資料集數量有誤

### DISCUSSION THREAD

**Discussion by szyitsai** [depositar-io/ckanext-data-depositario#13] [General]
https://data.depositar.io/dashboard/organizations
我的專案裡的資料集數量有誤
此專案的資料集數應該為23個，但顯示為0

**Comment by allycat** [comment]
Hey  我們已確認此狀況，會把它列為 issue ，在調查中。：）

**Reply by allycat** [reply]
謝謝回報，後台設定上的問題，目前已修正囉～　麻煩確認後，我就關閉此討論串了～

**Reply by szyitsai** [reply]
確認過沒問題了！
